moderation/Slowmode.go.tmpl

```diff
@@ -0,0 +1,41 @@
+{{/*
+	This commans helps using a slowmode. it deletes a user's message if their slowmode isnt over people who you want can bypass slowmode as well as set it 
+ 
+	Usage: "-slowmode on (time in seconds) / -slowmode off "
+ 
+	Recommended trigger: ".*"
+	Trigger type: Regex
+ 
+	Credits:
+	Sponge :D
+*/}}
+ 
+{{/* CONFIGURATION VALUES START */}}
+{{$bypassperms := "ManageMessages"}}
+{{$usageperms := "ManageMessages"}}
+{{/* put in the permissions you want the use to have in order to use or bypass the command */}}
+{{/* available perms: Administrator, ManageServer, ReadMessages, SendMessages, SendTTSMessages, ManageMessages, EmbedLinks, AttachFiles, ReadMessageHistory, MentionEveryone, VoiceConnect, VoiceSpeak, VoiceMuteMembers, VoiceDeafenMembers, VoiceMoveMembers, VoiceUseVAD, ManageNicknames, ManageRoles, ManageWebhooks, ManageEmojis, CreateInstantInvite, KickMembers, BanMembers, ManageChannels, AddReactions, ViewAuditLogs  */}}
+{{/* CONFIGURATION VALUES END */}}
+ 
+{{/* ACTUAL CODE */}}
+{{ dbSet 59 "bypass" $bypassperms }}
+{{ if (in (split (index (split (exec "viewperms") "\n") 2) ", ") $usageperms) }}
```

Comment:
You're using this `(in (split (index (split (exec "viewperms") "\n") 2) ", ")` pattern twice in this code, please extract it to a variable.